crates/schema/src/def.rs

```diff
@@ -280,58 +279,6 @@ impl ModuleDef {
         Some(TableSchema::from_module_def(self, table_def, (), table_id))
     }
 
-    /// Generate indexes for the module definition.
-    /// We guarantee that all `unique` constraints have an index generated for them.
-    /// This will be removed once another enforcement mechanism is implemented.
-    /// This is a noop if there are already usable indexes present.
-    fn generate_indexes(&mut self) {
```

Comment:
Is there info available in macros that isn't available here? If not, it may be easier to implement the patch here. Then we don't need to patch both C# and Rust macros to do this change.

(Unless this code was totally dead? If so my b.)

Reply:
On the other hand, we'll also need to patch the C# module library to support these.